harness/features/allFeatures.cloud.test.ts

```diff
@@ -0,0 +1,96 @@
+import { 
+    getConnectionStringForProxy, 
+    forEachSDK, 
+    describeIf, 
+    createClient, 
+    createUser,
+    callAllFeatures
+} from '../helpers'
+import { Capabilities, SDKCapabilities } from '../types'
+import { v4 as uuidv4 } from 'uuid'
+import { getServerScope } from '../mockServer'
+import { expectedFeaturesVariationOn } from '../mockData'
+
+jest.setTimeout(10000)
+
+const scope = getServerScope()
+
+describe('allFeatures Tests - Cloud', () => {
+    forEachSDK((name) => {
+        let url: string
+        const capabilities: string[] = SDKCapabilities[name]
+        const clientId: string = uuidv4()
+        const mockServerUrl 
+            = `http://${process.env.DOCKER_HOST_IP ?? 'host.docker.internal'}:${global.__MOCK_SERVER_PORT__}`
+
+        let variationOnUser: string
+        let invalidUser: string
+    
+        beforeAll(async () => {
+            url = getConnectionStringForProxy(name)  
+            // create client          
+            await createClient(url, clientId, 'dvc_server_test_token', { 
+                baseURLOverride: `${mockServerUrl}/client/${clientId}`, 
+                enableCloudBucketing: true,
+            })
+    
+            // create users
+            variationOnUser = (
+                await createUser(url, { user_id: 'user1', customData: { 'should-bucket': true } })
+            ).headers.get('location')
+    
+            invalidUser = (
+                await createUser(url, { name: 'invalid' })
+            ).headers.get('location')
+        })
+
+        describeIf(capabilities.includes(Capabilities.cloud))(name, () => {
+            it('should return all features without edgeDB', async () => {
+                scope
+                    .post(`/client/${clientId}/v1/features`, (body) => body.user_id === 'user1')
+                    .matchHeader('Content-Type', 'application/json')
+                    .matchHeader('authorization', 'dvc_server_test_token')
+                    .query((queryObj) => {
+                        return !queryObj.enableEdgeDB
+                    })
+                    .reply(200, expectedFeaturesVariationOn)
+                const featuresResponse = await callAllFeatures(clientId, url, variationOnUser)
+                
+                expect(scope.isDone()).toBeTruthy()
+                const features = (await featuresResponse.json()).data
+                expect(features).toMatchObject(expectedFeaturesVariationOn)
+            })
+
+            it('should return all features with edgeDB', async () => {
+                const newClientId = uuidv4()
+                const newClientResponse = await createClient(url, newClientId, 'dvc_server_test_token', { 
+                    baseURLOverride: `${mockServerUrl}/client/${newClientId}`,
+                    enableCloudBucketing: true,
+                    enableEdgeDB: true
+                })
+                await newClientResponse.json()
```

Comment:
I think you can remove this line since you're not storing the result in anything. There's a few other places that it can be removed too

Reply:
oh yeah good point, missed those